libcloud/dns/drivers/cloudflare.py

```diff
@@ -141,290 +159,252 @@ class CloudFlareDNSDriver(DNSDriver):
         RecordType.URL: 'LOC'
     }
 
+    PAGE_SIZE = 20
```

Comment:
Does CloudFlare API support larger page size? If so, using something larger (e.g. 100) might be more efficient in some scenarios for accounts with a lot of domains / records.

Reply:
Good idea. I've increased the page size to the API maximum (instead of the previous API default value) in 9970385.